MetaMask/metamask-extension
# Enable blockaid in main build

## **Description**
Enable blockaid in main build.

## **Manual testing steps**

_1. Run extension locally using `yarn start`
_2. Go to test dapp and execute any PPOM transaction
_3. Check for presence of alert on confirmation page


## **Related issues**

_Fixes MetaMask/MetaMask-planning#1351

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
